packages/form-builder/addon/components/cfb-form-editor/show-advanced-btn.hbs

```diff
@@ -0,0 +1,12 @@
+<UkButton
+  @color="link"
+  @onClick={{@onClick}}
+  class="uk-flex uk-flex-middle uk-margin"
+>
+  {{#if @showAdvanced}}
+    <UkIcon @icon="triangle-down" />
+  {{else}}
+    <UkIcon @icon="triangle-right" />
+  {{/if}}
+  {{t "caluma.form-builder.question.advancedSettings"}}
+</UkButton>
```

Comment:
Let's implement this as wrapper component. So the call site would be something like:

```hbs
<CfbAdvancedSettings>
  my advanced settings...
</CfbAdvancedSettings>
```

This way the component using it must not implement any toggling and toggle state.

Reply:
yh i was thinking about that. I changed it.

Reply:
Nice! But let's name it something like `CfbAdvancedSettings` as it isn't just a button anymore.